flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/BuiltInAggregateFunctionTestBase.java

```diff
@@ -344,6 +361,38 @@ public void execute(TableEnvironment tEnv, Table sourceTable) {
         protected abstract TableResult getResult(TableEnvironment tEnv, Table sourceTable);
     }
 
+    private abstract static class RuntimeFailureItem implements TestItem {
```

Comment:
call it `RuntimeErrorItem` to be in sync with subclasses

Reply:
I followed the `SuccessItem` convention. `Success/Failure`, but sure I'll rename it.